webapp-src/src/Login/SelectAccount.js

```diff
@@ -29,17 +29,19 @@ class SelectAccount extends Component {
   }
   
   handleSelectAccount() {
```

Comment:
I like the new behavior, more simple and intuitive, but since the `Select` button has been replaced by a `Back` button, this handle should be renamed `handleBack`.